agent/check_handler_internal_test.go

```diff
@@ -133,7 +134,10 @@ func TestExecuteCheck(t *testing.T) {
 	assert.NoError(json.Unmarshal(msg.Payload, event))
 	assert.NotZero(event.Timestamp)
 	assert.True(event.HasMetrics())
-	assert.Equal(2, len(event.Metrics.Points))
```

Comment:
require.Equal(2, len(event.Metrics.Points), string(event)) should print it out in case of failure

Reply:
ahh 😅 that is helpful! `msgAndArgs ...interface{}`